content/workshops/cascading-styles-deep.md

```diff
@@ -17,71 +21,120 @@ tags:
+- The advantages of logical vs physical dimensions.
+- The layout flow with intrinsic and extrinsic sizing.
+- Size comparison functions like min(), max(), and clamp().
+- When to use flexbox vs grid for layout and alignment.
+- Fast and dynamic layouts using subgrid and container queries.
+
+## Who Is This For?
+
+This workshop is intended for designers & developers
+with at least a basic understanding in HTML and CSS.
+You don’t need to be an expert to keep up,
+but even the experts are likely to learn something new.
+
+## Schedule
+
+The workshop is taught in five sessions, each with two parts:
+
+### 1. Resilient Styles: a Declarative Cascade
+
+#### First, Do No Harm
+
+CSS is fundamentally different from other languages or design tools, built around a radical vision for contextual style and user-control. We’ll dig into the practical implications of that vision, and how the ‘grain’ of the language can guide us to more performant and resilient styles.
+
+#### Cascading & Inheritance
+
+A deep-dive into the algorithms that take us from simple property/value declarations to a fully-styled web application. Along the way, we’ll explore new features like nesting, cascade layers, scope, and the :has() selector.
+
+### 2. Dynamic Systems: Custom Properties & Value Resolution
+
+#### Custom Properties Reveal the Matrix
+
+CSS custom properties (aka “variables”) expose the internals of CSS value resolution and error recovery. What does it even mean for properties to become ‘invalid at computed value time’? And how can we use these CSS internals to our advantage, developing more robust and dynamic style systems?
+
+#### CSS Variables in Practice
+Practical use-cases, and interactive exercises related to CSS variables and functions…
+
+### 3. Intrinsic Layouts: Distributing Space
+
+#### Flowing & Flexing
+
+Unlike the printed page, web content and context can be unpredictable. CSS provides tools to manage that uncertainty, aligning & distributing objects on the page. We’ll talk about normal flow, intrinsic and extrinsic sizing, logical properties, box sizing & alignment, and the flexible box model.
+
+#### Flex & Alignment in Practice
+
+Practical use-cases, and interactive exercises related to distributing space…
+
+### 4. Intrinsic Layouts: Defining Structure
+
+#### Defining Structure
+
+Sometimes we also need to impose external structure to create consistent and reliable layouts, even with unpredictable content. This is a full session dedicated to Grids & Sub-grids, the multiple ‘stages of squishiness’, and container queries.
+
+#### Grids & Containers in Practice
+
+Practical use-cases, and interactive exercises related to defining layouts…
+
+### 5. Resilient Styles: Organizing Conventions
+
+#### Cascade Aligned Programming
+
+There are many conventions for organizing CSS, but the best systems all share underlying principles based on CSS itself. What would it mean to use The Cascade as our guide for writing maintainable CSS?
```

Comment:
does "The Cascade" need to be capitalized here? Maybe we should use <em> if we want to highlight this?